libs/core/src/lib/nested-list/nested-item/nested-item.directive.ts

```diff
@@ -107,14 +131,18 @@ export class NestedItemDirective implements AfterContentInit, NestedItemInterfac
 
     /** Method that dispatches `click` event on link item*/
     click(): void {
-        if (this.linkItem) {
+        if (this.contentItem) {
+            this.contentItem.click();
+        } else if (this.linkItem) {
```

Comment:
is it possible that this.linkItem won't exist? If not then maybe this function would be better
`
click(): void {
if(this.contentItem){
  this.contentItem.click();
return;
}
this.linkItem.focus();
}
`

Reply:
It is possible, user decides about the markup, so he can put whatever he wants.